pipelines/toast_ground_schedule.py

```diff
@@ -324,7 +326,7 @@ def update(self, *args, **kwargs):
 
 
 class SSOPatch(Patch):
-    def __init__(self, name, weight, radius, el_min, el_max):
+    def __init__(self, name, weight, radius):
```

Comment:
Looks like the SSOPatch class got truncated. Perhaps your copy of TOAST wasn't quite up-to-date? Can you please rebase the branch so we don't loose any recent changes?